vrt/config.js

```diff
@@ -340,6 +340,9 @@ const config = {
       },
     ],
   },
+  'toaster-focus': {
+    skip: true,
```

Comment:
that's the reason for skipping it?

Reply:
Since I added an e2e test for functionality I didn't think we needed visual coverage.